cypress/e2e/templates.spec.js

```diff
@@ -2,7 +2,8 @@
  * SPDX-FileCopyrightText: 2023 Julius Härtl <jus@bitgrid.net>
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
-describe('Create new office files from templates', function() {
+
+describe.skip('Create new office files from templates', function() {
```

Comment:
```suggestion
describe('Create new office files from templates', function() {
```

Reply:
Thanks for pointing this out, forgot to remove it :)